Assets/Scripts/Menu Scripts/SensorManager.cs

```diff
@@ -13,9 +13,10 @@ public class SensorManager : MonoBehaviour {
     public Button rightButton;
     public Text sensorText;
     public GameObject togglePrefab;
+    public GameObject sensorMenu;
 
     private List<ROSSensorConnectionInterface> sensorList = new List<ROSSensorConnectionInterface>();
-    private List<string> subscriberList = new List<string>();
+    private Dictionary<string, bool> subscriberDict = new Dictionary<string, bool>();
```

Comment:
what is the purpose of the bool?

Reply:
to store the status of the subscription. if it's subscribed or not currently.